README.md

```diff
@@ -40,3 +40,49 @@ interceptor := grpc.UnaryInterceptor(
     }),
 )
 ```
+
+### Logtracing
+
+Used with [tags](`github.com/grpc-ecosystem/go-grpc-middleware/tags`) and [logging](`github.com/grpc-ecosystem/go-grpc-middleware/logging`), allows you to log out a trace and span ID for each call.
```

Comment:
I would be slightly more precise in the language so you can also explain how it works. I.e. adds the span_id and trace_id to the fields that are logged. I would also specify with which tracing system this works.